### DISCUSSION TITLE
Adding a timestamp to the vulnerability report

### DISCUSSION THREAD

**Discussion by pablogalegoc** [aquasecurity/trivy#5536] [Ideas]
### Description

It would be nice if the vulnerability reports had a timestamp with the creation date of the document. Something at the top level like:
```json
{
  "SchemaVersion": 2,
  "Timestamp": "2023-11-01T16:56:23Z",
  "ArtifactName": "library/redis",
  "ArtifactType": "container_image",
  "Metadata": {
```

### Target

None

### Scanner

Vulnerability

**Comment by knqyf263** [comment]
Thanks. Created https://github.com/aquasecurity/trivy/issues/5542
